### ISSUE TITLE
Implement flashing AP bar (for not enough to move)

### ISSUE THREAD

**Issue by jabelch** [akintu/akintu#108]

**Comment by jabelch** [comment]
So Devin pointed out that I didn't have enough AP to move again because of my character's AP level (moving didn't evenly divide into my AP).  However, this is an important issue - we should have some type of visual feedback (scrolling message, flashing AP bar are some ideas) to indicate that you can't move, so you should end your turn with "n".  This would be nice to have in the side bar.

**Comment by basepi** [comment]
Erm, why am I assigned to this one?

**Comment by BlackVegetable** [comment]
To indicate you have AP less than the movement cost would require a "flashing bar" or some other UI component.

**Comment by basepi** [comment]
Oh, that's what I get for not reading through the entire issue.  I would think you'd just print to the text area that you don't have enough AP.  But I could implement an API for you to make a bar flash.

**Comment by BlackVegetable** [comment]
Yeah, I'm sending enough messages to the text area as it is.  I think a flashing bar would be a nice touch.

**Comment by basepi** [comment]
Kk, I'll look into it next week.

**Comment by BlackVegetable** [comment]
@basepi this is way overdue. What have you been doing for four years that was more important than this?

/trololololol

**Comment by basepi** [comment]
Well hello there blast from the past.
